.github/workflows/push.yml

```diff
@@ -423,7 +423,7 @@ jobs:
           TAG: ${{ needs.set-version.outputs.tag }}
         run: |
           hub release download "${TAG}" -i '*windows_amd64*.zip'
-          printf "zip=%s\n" *.zip >> $GITHUB_OUTPUT
+          zipFile=$(basename *.zip) && echo "zip=$zipFile" >> $GITHUB_OUTPUT
```

Comment:
Tohle jsem asi z toho `set-output` přepsal blbě protože failuje nějak divně build msi níže. Ozkoušel jsem a teď by to mělo fungovat. 🙂
![CleanShot 2022-10-14 at 18 07 00](https://user-images.githubusercontent.com/215660/195891848-25e3c643-e890-4f31-a6a0-8daad300e631.jpg)

Reply:
Myslím si tak naivně, že by se do toho `$GITHUB_OUTPUT` neměla posílat ta newline. Jinak se to chová stejně a jinak mě nenapadá co by mohlo být blbě. 😕